_ml-commons-plugin/model-access-control.md

```diff
@@ -0,0 +1,597 @@
+            "ignore_unmapped": false,
+            "score_mode": "none",
+            "boost": 1
+          }
+        }
+      ]
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+#### Example response
+
+```json
+{
+  "took": 6,
+  "timed_out": false,
+  "_shards": {
+    "total": 1,
+    "successful": 1,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": {
+    "total": {
+      "value": 4,
+      "relation": "eq"
+    },
+    "max_score": 0,
+    "hits": [
+      {
+        "_index": ".plugins-ml-model-group",
+        "_id": "TRqZfYgBD7s2oEFdvrQj",
+        "_version": 1,
+        "_seq_no": 2,
+        "_primary_term": 1,
+        "_score": 0,
+        "_source": {
+          "backend_roles": [
+            "HR",
+            "IT"
+          ],
+          "owner": {
+            "backend_roles": [
+              "HR",
+              "IT"
+            ],
+            "custom_attribute_names": [],
+            "roles": [
+              "ml_full_access",
+              "own_index",
+              "test_ml"
+            ],
+            "name": "user1",
+            "user_requested_tenant": "__user__"
+          },
+          "created_time": 1685734407714,
+          "access": "restricted",
+          "latest_version": 0,
+          "last_updated_time": 1685734407714,
+          "name": "model_group_test",
+          "description": "This is an example description"
+        }
+      },
+      ...
+    ]
+  }
+}
+```
+
+#### Example request: Search for model groups with a model group ID
+
+```json
+GET /_plugins/_ml/model_groups/_search
+{
+  "query": {
+    "bool": {
+      "must": [
+        {
+          "terms": {
+            "_id": [
+              "HyPNK4gBwNxGowI0AtDk"
+            ]
+          }
+        }
+      ]
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+#### Example response
+
+```json
+{
+  "took": 2,
+  "timed_out": false,
+  "_shards": {
+    "total": 1,
+    "successful": 1,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": {
+    "total": {
+      "value": 1,
+      "relation": "eq"
+    },
+    "max_score": 1,
+    "hits": [
+      {
+        "_index": ".plugins-ml-model-group",
+        "_id": "HyPNK4gBwNxGowI0AtDk",
+        "_version": 3,
+        "_seq_no": 16,
+        "_primary_term": 5,
+        "_score": 1,
+        "_source": {
+          "backend_roles": [
+            "IT"
+          ],
+          "owner": {
+            "backend_roles": [
+              "",
+              "HR",
+              "IT"
+            ],
+            "custom_attribute_names": [],
+            "roles": [
+              "ml_full_access",
+              "own_index",
+              "test-ml"
+            ],
+            "name": "user1",
+            "user_requested_tenant": null
+          },
+          "created_time": 1684362035938,
+          "latest_version": 2,
+          "last_updated_time": 1684362571300,
+          "name": "model_group_test",
+          "description": "This is an example description",
+          "tags": {
```

Comment:
Can we remove the tags fields and the keys from this response? We actually removed this field and this happened to be an old response.